django_extensions/auth/mixins.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.mixins import AccessMixin
+
```

Comment:
The build is failing on a very minor whitespace thing here :-) 
`pep8` says top level functions and classes should be separated with two blank lines instead of one.